src/views/ModelPlan/Collaborators/table.tsx

```diff
@@ -75,7 +75,10 @@ const CollaboratorsTable = ({
       {
         Header: collaboratorsMiscT('table.actions'),
         Cell: ({ row }: any) => {
-          if (row.original.teamRole === 'MODEL_LEAD' && isLastLead) {
+          if (
+            row.original.teamRoles.includes(TeamRole.MODEL_LEAD) &&
+            isLastLead
+          ) {
```

Comment:
This code addresses removing the `Edit` and `Remove` links to the only `Model Lead` in the collaborator list